vaccine_feed_ingest/runners/co/colorado_gov/parse.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python
+
+import json
+import pathlib
+import sys
+
+from lxml import etree
+from pykml import parser
+
+
+def parse_point(element):
+    coords = element.coordinates.text.strip().split(",")
+    return {"lat": coords[0], "long": coords[1]}
+
+
+def parse_extended_data(element):
+    result = {}
+    for data in element.Data:
+        result[data.attrib["name"]] = data.value.text
+    return result
+
+
+element_parsers = {
+    "Point": parse_point,
+    "ExtendedData": parse_extended_data,
+}
+
+
+def parse(input_file):
+    with input_file.open("r") as f:
+        contents = f.read().encode("UTF-8")
+        root = parser.fromstring(contents)
+
+    locations = []
+    for folder in root.Document.Folder:
+        for place in folder.Placemark:
+            location = {"_folder_name": folder.name.text}
+            for item in place.getchildren():
+                key = etree.QName(item).localname
+                if key in element_parsers:
+                    location[key] = element_parsers[key](item)
+                else:
+                    # if key starts with an uppercase letter, there's probably
+                    # a element to be parsed more that just '.text'
+                    if key[0].isupper():
+                        print(f"WARNING: {key} should probably have a parser")
```

Comment:
nit: could you swap this to a `logger` (check other runners for an example) rather than a `printf`?

Reply:
Thanks for catching this, it was left over from development. I've noticed before that the logger doesn't seem to output anything, I assume I just haven't figured out how to set the log level. Is there an example somewhere of how to do this?

Reply:
You're right! Check [this normalizer](https://github.com/CAVaccineInventory/vaccine-feed-ingest/blob/main/vaccine_feed_ingest/runners/ak/arcgis/normalize.py) for a sample configuration (you can probably copy/paste it)

Reply:
Thanks for the example, there's several places that are missing logger configuration, so I intend to leave this for now and open a separate PR to address missing config more broadly